dataactbroker/handlers/accountHandler.py

```diff
@@ -258,20 +258,20 @@ def perms_to_affiliations(perms):
             continue
 
         codes, perm_level = components
-        perm_level = perm_level.lower()
         split_codes = codes.split('-FREC_')
         frec_code = cgac_code = None
-        if len(split_codes) == 2:
+        if len(split_codes) == 2:  # permissions for FR entity code
             frec_code = split_codes[1]
             if frec_code not in available_frecs:
                 logger.warning('Malformed permission: %s', perm)
                 continue
-        else:
+        else:  # permissions for CGAC
```

Comment:
We try not to do comments in the same line as the code where possible. Move these above the `if` and `else`